AgileBoard/src/AgileBoard.Api/Program.cs

```diff
@@ -0,0 +1,46 @@
+using AgileBoard.Application.ExceptionHandling;
+using AgileBoard.Core.Installers;
+using AgileBoard.Infrastructure.Extensions;
+using AgileBoard.Application.Extensions;
+using AgileBoard.Core.Extensions;
+using AgileBoard.Infrastructure.Time;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services
+    .AddControllers()
+    .AddNewtonsoftJson();
+
+builder.Services
+    .AddApplication()
+    .AddCore();
+
+builder.AddInfrastructre();
+
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+builder.InstallServicesInAssembly();
+builder.Services.AddAutoMapper(AppDomain.CurrentDomain.GetAssemblies());
+
+builder.Services.AddTransient<Clock, Clock>();
```

Comment:
Why Transient? it only returns time, it can be actually singleton or I'd prefer to do that scoped